src/Nfield.Quota/QuotaFrameValidator.cs

```diff
@@ -137,12 +137,17 @@ public QuotaFrameValidator()
             ICollection<QuotaVariableDefinition> varDefinitions,
             PropertyValidatorContext context)
         {
-            var expression = new Regex("^[a-zA-Z][a-zA-Z0-9_]*$");
+            // don't allow unicode whitespace or special characters
+            var invalidExpression = new Regex(
+                @"^[^ \u00A0\u1680​\u180e\u2000-\u2009\u200a​\u200b​\u202f\u205f​\u3000.,\/#!$%\^&\*;:{}=\-`~()@<>|'""\+\\\[\]\?]+$");
```

Comment:
Same comments as on the client: lets at least add a source where this statement is coming from so people can read about it and get more context :)